docs/en/installing.rst

```diff
@@ -85,8 +85,19 @@ PlatformIO can be used in two flavors:
 
 To install PlatformIO, you can follow this Getting Started, provided at `docs.platformio.org`_.
 
-Using the stable code
-*********************
+To get more information about PlatformIO, see the following links:
```

Comment:
To avoid "To" twice, I recommend starting the second sentence with "For more information about...".

Reply:
Thx. Changed